### ISSUE TITLE
Model weights

### ISSUE THREAD

**Issue by tforgaard** [zhangq327/U-MAE#10]
Hi!
Thank you for your inspiring work!

Would it be possible to get access to the weights of the models pretrained with the uniformity loss?

Thanks again

**Comment by zhangq327** [comment]
HI!
Thanks for appreciating our work. We would like to release the weights of the pre-trained models. However, the experiments in this paper are conducted on a cluster that forbids downloading checkpoints to the local computers or uploading checkpoints to online drives. So we recommend that you can train the model by yourself. If you encounter any problems, feel free to contact me at zhangq327@stu.pku.edu.cn or open a new issue.

Hope it helps!
